Samples/NetworkMediaStreamer/RTPMediaStreamer/src/RTPStreamSink.cpp

```diff
@@ -0,0 +1,433 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+#include <pch.h>
+
+TxContext::TxContext(std::string destination, winrt::PacketHandler packetHandler /*= nullptr*/)
+    : m_u64StartTime(0)
+    , m_packetHandler(packetHandler)
+    , m_ssrc(0)
+    , m_RtpSocket(INVALID_SOCKET)
+    , m_RtcpSocket(INVALID_SOCKET)
+    , m_localRTPPort(0)
+    , m_localRTCPPort(0)
+    , m_remotePort(0)
+    , m_SequenceNumber(0)
+{
+    memset(&m_remoteAddr, 0, sizeof(m_remoteAddr));
+    std::string param = "ssrc=";
+    auto sep2 = destination.find(param);
+    if (sep2 != std::string::npos)
+    {
+        auto sep3 = destination.find("&", sep2);

```

Comment:
>sep3 [](start = 13, length = 4)

what happens if not found?

Reply:
substr handles this case and returns till end of the string.
but i shall add the check anyway

---
In reply to: [491169880](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491169880) [](ancestors = 491169880)